src/frontend/components/UI/Sidebar/components/CurrentDownload/index.tsx

```diff
@@ -52,37 +52,35 @@ export default React.memo(function CurrentDownload({ appName, runner }: Props) {
   }
 
   return (
-    <>
-      <Link to={`/download-manager`} className="currentDownload">
-        <span className="statusIcon" title={`${getStatus()} - ${gameTitle}`}>
-          <Badge
-            badgeContent={`${Math.round(progress.percent ?? 0)}%`}
-            color="primary"
-          >
-            <FontAwesomeIcon icon={faDownload} />
-          </Badge>
-        </span>
+    <Link to={`/download-manager`} className="currentDownload">
+      <span className="statusIcon" title={`${getStatus()} - ${gameTitle}`}>
+        <Badge
+          badgeContent={`${Math.round(progress.percent ?? 0)}%`}
+          color="primary"
+        >
+          <FontAwesomeIcon icon={faDownload} />
+        </Badge>
+      </span>
 
-        <div className="full-size">
-          <span className="gameTitle">{gameTitle ?? 'GameName'}</span>
-          <br />
-          <span className="downloadStatus">{getStatus()}</span>
-          <br />
-          <Box sx={{ display: 'flex', alignItems: 'center' }}>
-            <Box sx={{ width: '100%', mr: 1 }}>
-              <LinearProgress
-                variant="determinate"
-                value={progress.percent || 0}
-              />
-            </Box>
-            <Box sx={{ minWidth: 35 }}>
-              <Typography variant="body2">{`${Math.round(
-                progress.percent || 0
-              )}%`}</Typography>
-            </Box>
+      <div className="full-size">
+        <span className="gameTitle">{gameTitle ?? 'GameName'}</span>
+        <br />
+        <span className="downloadStatus">{getStatus()}</span>
+        <br />
+        <Box sx={{ display: 'flex', alignItems: 'center' }}>
+          <Box sx={{ width: '100%', mr: 1 }}>
+            <LinearProgress
+              variant="determinate"
+              value={progress.percent || 0}
+            />
           </Box>
-        </div>
-      </Link>
-    </>
+          <Box sx={{ minWidth: 35 }}>
+            <Typography variant="body2">{`${Math.round(
+              progress.percent || 0
+            )}%`}</Typography>
+          </Box>
+        </Box>
+      </div>
+    </Link>
```

Comment:
This is just me removing the extra `<>` fragment that was not needed, the diff is just the indentation